nixos/tests/mpd.nix

```diff
@@ -0,0 +1,118 @@
+import ./make-test.nix ({ pkgs, ... }:
+  let
+    track = pkgs.fetchurl {
+      # Sourced from http://freemusicarchive.org/music/Blue_Wave_Theory/Surf_Music_Month_Challenge/Skyhawk_Beach_fade_in
+      # License: http://creativecommons.org/licenses/by-sa/4.0/
+
+      name = "Blue_Wave_Theory-Skyhawk_Beach.mp3";
+      url = https://freemusicarchive.org/music/download/c7f57c4f783d3226610aaa41c5c2f0c7cbbb2958;
```

Comment:
This isn't a persistent url, use `https://freemusicarchive.org/file/music/ccCommunity/Blue_Wave_Theory/Surf_Music_Month_Challenge/Blue_Wave_Theory_-_04_-_Skyhawk_Beach.mp3` instead

Reply:
Yes, that's the polkit issue I mentioned (or something that's affecting polkit). I'll just rebase this unto the current nixos-unstable branch. It will run there.

Reply:
Ohh right, sorry